src/odb/include/odb/db.h

```diff
@@ -7195,9 +7195,12 @@ class dbTechLayer : public dbObject
   int getPitch();
   int getPitchX();
   int getPitchY();
+  int getFirstLastPitch();
   void setPitch(int pitch);
   void setPitchXY(int pitch_x, int pitch_y);
+  void setFirstLastPitch(int first_last_pitch);
   bool hasXYPitch();
+  bool hasPitch();
```

Comment:
Why is this need for this change?

Reply:
It is used to see if the pitch has alredy been set by the LEF58_PITCH, if it has then the PUTCH is ignored.

Reply:
The PITCH statement should precede the LEF58_PITCH so I don't think this is necessary.  

Its a bit confusing as an API as the PITCH will always be set when parsing is finished.

Reply:
I'm confused about how this happens.  The PITCH statement should come before the LEF58_PITCH.  Does the parser not return these in order?

Reply:
I belive when it first reads the LEF the parser reads it in order and store the information on the class lefiLayer, but it doesn't fineshes parsing the LEF58 properties. So on the class lefiLayer it first fineshes parsing the LEF58 properties and tranfering the information to the dbTechLayer, and after that it transfers the information already parsed from the lefiLayer to the dbTechLayer.